tests/data/test_AtomicData.py

```diff
@@ -19,6 +19,30 @@ def test_from_ase(CuFcc):
         assert data[key].shape == (len(atoms), 3)  # 4 species in this atoms
 
 
+def test_to_ase(CH3CHO):
+    atoms, data = CH3CHO
+    to_ase_atoms = data.to_ase()
+    assert np.allclose(atoms.get_positions(), to_ase_atoms.get_positions())
+    assert np.array_equal(atoms.get_atomic_numbers(), to_ase_atoms.get_atomic_numbers())
+    assert np.array_equal(atoms.get_pbc(), to_ase_atoms.get_pbc())
+    assert np.array_equal(atoms.get_cell(), to_ase_atoms.get_cell())
+
+
+def test_to_ase_batches(atomic_batch):
+    atomic_data = AtomicData.from_dict(vars(atomic_batch))
```

Comment:
Why `vars`?

Reply:
For some reason `dict` gave an "attribute of type list is not callable" error

Reply:
try `a.to_dict()` instead of `dict(a)`